controllers/tests/controller/selfnoderemediation_controller_test.go

```diff
@@ -933,13 +866,15 @@ func clearEvents() {
 }
 
 func verifyEvent(eventType, reason, message string) {
-	isEventMatch := isEventOccurred(eventType, reason, message)
-	ExpectWithOffset(1, isEventMatch).To(BeTrue())
+	EventuallyWithOffset(1, func() bool {
+		return isEventOccurred(eventType, reason, message)
+	}, 5*time.Second, 250*time.Millisecond).Should(BeTrue())
 }
 
 func verifyNoEvent(eventType, reason, message string) {
-	isEventMatch := isEventOccurred(eventType, reason, message)
-	ExpectWithOffset(1, isEventMatch).To(BeFalse())
+	EventuallyWithOffset(1, func() bool {
```

Comment:
this should be Consistently I guess?

Reply:
Nice catch, thanks.

Reply:
Here: https://github.com/medik8s/self-node-remediation/pull/178